erpnext/hr/doctype/employee/employee.py

```diff
@@ -254,15 +254,18 @@ def send_birthday_reminders():
 	if int(frappe.db.get_single_value("HR Settings", "stop_birthday_reminders") or 0):
 		return
 	employees = get_employees_born_today()
-	if employees:
-		recipients_list = frappe.get_all('Employee', filters={'status': 'Active'})
-		recipients = get_employee_emails(recipients_list)
+	companies_with_birthdays = set(employee['company'] for employee in employees)
 
+	if employees:
+		recipients_list = frappe.get_all('Employee', fields = ['name', 'company'], filters = {'status': 'Active', 'company': ('in', companies_with_birthdays)})
 		birthday_email_template = frappe.db.get_single_value("HR Settings", "birthday_email_template")
 		if birthday_email_template:
 			email_template = frappe.get_doc("Email Template", birthday_email_template)
 
 		for employee in employees:
+			recipients = [{'name': recipient['name']} for recipient in recipients_list if recipient['company'] == employee.company ]
```

Comment:
Use `dict.get` here as well, and do we need a pass a dictionary to `get_employee_emails`? Wouldn't just the name do?